### PR TITLE
Add WriteHWRegRepeat

### PR DESCRIPTION
**Pull request by MoochMcGee**
This hasn't been tested at all. I figured it was low-hanging fruit tho.

### CODE REVIEW

**Comment by Subv** [comment]
Isn't this supposed to "repeatedly write the register"? I'm not sure what that means though

**Comment by MoochMcGee** [comment]
OH WHOOPS

File: src/core/hle/service/gsp_gpu.cpp [hunk 1]
```diff
@@ -167,6 +167,29 @@ static void WriteHWRegsWithMask(Service::Interface* self) {
...
+    //Taking size to mean amount of times the reg is hammered.
+    /*// size should be word-aligned
+    if ((size % 4) != 0) {
+        LOG_ERROR(Service_GSP, "Invalid size 0x%08x", size);
+        return;
+    }*/
+    
+    u32 data = cmd_buff[4];
+    
+    for(int i = 0;i<size;i++) GPU::Write<u32>(reg_addr,data);
```
Comment (lioncash) [inline src/core/hle/service/gsp_gpu.cpp]:
```cpp
for (int i = 0; i < size; i++) 
    GPU::Write<u32>(reg_addr, data);
```

File: src/core/hle/service/gsp_gpu.cpp [hunk 2]
```diff
@@ -167,6 +167,29 @@ static void WriteHWRegsWithMask(Service::Interface* self) {
...
+        LOG_ERROR(Service_GSP, "Write address out of range! (address=0x%08x, size=0x%08x)", reg_addr, size);
+        return;
+    }
+    
+    //Taking size to mean amount of times the reg is hammered.
+    /*// size should be word-aligned
+    if ((size % 4) != 0) {
+        LOG_ERROR(Service_GSP, "Invalid size 0x%08x", size);
+        return;
+    }*/
```
Comment (lioncash) [inline src/core/hle/service/gsp_gpu.cpp]: If this isn't gong to be used, then just remove it entirely. Don't leave stuff like this around without an explanation.

**Comment by Subv** [comment]
I don't think it should work like this, this needs a hwtest

**Comment by MoochMcGee** [comment]
Yeah, good point.

**Pull request closed without merging**